src/drt/src/pa/FlexPA_unique.h

```diff
@@ -96,18 +96,18 @@ class UniqueInsts
   // All the unique instances
   std::vector<frInst*> unique_;
   // Mapp all instances to their representative unique instance
-  std::map<frInst*, frInst*, frBlockObjectComp> inst2unique_;
+  std::map<frInst*, frInst*, frBlockObjectComp> inst_to_unique_;
   // Maps all instances to the set of instances with the same unique inst
-  std::unordered_map<frInst*, InstSet*> inst2Class_;
+  std::unordered_map<frInst*, InstSet*> inst_to_class_;
   // Maps a unique instance to its pin access index
-  std::map<frInst*, int, frBlockObjectComp> unique2paidx_;
+  std::map<frInst*, int, frBlockObjectComp> unique_to_pa_idx_;
   // Maps a unique instance to its index in unique_
-  std::map<frInst*, int, frBlockObjectComp> unique2Idx_;
+  std::map<frInst*, int, frBlockObjectComp> unique_to_idx_;
   // master orient track-offset to instances
   std::map<frMaster*,
            std::map<dbOrientType, std::map<std::vector<frCoord>, InstSet>>,
            frBlockObjectComp>
-      masterOT2Insts_;
+      master_OT_to_insts_;
```

Comment:
What is OT?

Reply:
I think its Orient Type (`dbOrientType`) but I'm not completely sure what its supposed to mean

Reply:
From `masterOT2Insts_[inst->getMaster()][orient][offset].insert(inst.get());`  I would guess master_orient_trackoffset_to_insts_ is intended (maybe O=orient T=track offset?).

Reply:
Oh, ok, that makes sense